webv2/table/utilities.go

```diff
@@ -335,3 +337,37 @@ func getSequenceId(sequenceName string, spSeq map[string]ddl.Sequence) string {
 	}
 	return ""
 }
+
+func UpdateDefaultValue(dv ddl.DefaultValue, tableId, colId string, conv *internal.Conv) {
```

Comment:
This code will be tested through the Update API and tests will be added there

Reply:
There are a couple of differences:
1. This API is only for default values but the `spannerSchemaApplyExpressions` will service for all expression related values such as generated columns, etc.
2. This is called as part of the Edit column functionality and updates the value without verification where as the `spannerSchemaApplyExpressions` also handles any errors when verification api is called
3. This handles update, add, delete for default values which that is just to applies the first round of default values

Reply:
Okay 👍 
Can you add a comment above both these comments? A small comment distinguishing them would be enough.

Reply:
Sure!